lib/parse.js

```diff
@@ -363,6 +364,8 @@ function isMultipleOf(value, step) {
   return Math.abs(n - nInt) < Math.pow(10, (`${nInt}`.length - 16));
 }
 
+const MAX_EDIT = 3;
```

Comment:
I think 1 is good, 2 should be fine, 3 is too much to be considered as a typo. @tophf

Reply:
Why is `3` too much? It's certainly possible to make 3 typos so ideally I would try to calculate the distance of the key on the keyboard, which unfortunately isn't universally usable as notebooks tend to have unique layouts.

Reply:
Note that this number applies to a single word. For example, the edit distance from `var` to `name` is also 3 (v -> n, r -> m, insert e).

Reply:
Can it be dynamic? Like `word.length / 2`?

Reply:
Dynamically is possible, but will need test-cases for it's own. Because you would be surprised how closely certain words are in terms of `Distance cost`.

Reply:
`log2` seems like a good average for the max amount of `edits`